servers/rendering/renderer_rd/shaders/effects/copy.glsl

```diff
@@ -245,9 +247,22 @@ void main() {
 #ifdef MODE_LINEARIZE_DEPTH_COPY
 
 	float depth = texelFetch(source_color, pos + params.section.xy, 0).r;
+	float z_near = params.camera_z_near;
+	float z_far = params.camera_z_far * params.camera_z_far_scale;
+
 	depth = depth * 2.0 - 1.0;
-	depth = 2.0 * params.camera_z_near * params.camera_z_far / (params.camera_z_far + params.camera_z_near - depth * (params.camera_z_far - params.camera_z_near));
-	vec4 color = vec4(depth / params.camera_z_far);
+#ifdef USE_ORTHOGONAL_PROJECTION
+	depth = ((depth + (z_far + z_near) / (z_far - z_near)) * (z_far - z_near)) / 2.0;
+#else
+	depth = 2.0 * z_near * z_far / (z_far + z_near - depth * (z_far - z_near));
+#endif
+	float depth_linear_normalized = depth / z_far;
+
+	vec4 color = vec4(depth_linear_normalized, depth_linear_normalized, depth_linear_normalized, 1.0);
+
+	if (depth >= z_far - 0.01) {
+		color = vec4(0.0, 0.2705, 0.5490, 1.0);
```

Comment:
Out of curiosity, was this shade of blue chosen arbitrarily or does it have special meaning?